### PR TITLE
Fix logit scale calculation in zero_shot_iid.py

### PR DESCRIPTION
**Pull request by work4cs**
https://github.com/Imageomics/bioclip/blob/4ea32471c550a1c42f21ff1c2d08e2064f8ac048/src/open_clip/model.py#L237

### CODE REVIEW

**Review by egrace479 — APPROVED** [review]
Back to how it had been ✅

**Comment by hlapp** [comment]
If this is the correct form, then we'll [need to fix this in pybioclip](https://github.com/Imageomics/pybioclip/blob/a96a21f918ee51397583c0b6e25db112512e7c02/src/bioclip/predict.py#L242) as well.

And presumably this doesn't affect ordering after `softmax()`? But it does affect the reported probabilities?

**Comment by work4cs** [comment]
> If this is the correct form, then we'll [need to fix this in pybioclip](https://github.com/Imageomics/pybioclip/blob/a96a21f918ee51397583c0b6e25db112512e7c02/src/bioclip/predict.py#L242) as well.
> 
> And presumably this doesn't affect ordering after `softmax()`? But it does affect the reported probabilities?

I am thinking of if double exp(), some values might become too large and overflow. Then, the ranking becomes meaningless and wrong. And probabilities/confidence, yes, it might make the result seems over-confident.

**Comment by egrace479** [comment]
@work4cs is going to explore this more over the weekend before we make any changes.

**Comment by hlapp** [comment]
So sounds like we do need to propagate this to pybioclip as well.
